rasa_nlu/featurizers/regex_featurizer.py

```diff
@@ -72,18 +72,19 @@ def features_for_patterns(self, message):
         relating the name of the regex to whether it was matched."""
 
         matches = []
```

Comment:
best is to rename this
```suggestion
        found_patterns = []
```